### ISSUE TITLE
`uses-rel-preconnect` reports "Audit did not produce a value at all. "maxLength" might not be a valid assertion for this audit."

### ISSUE THREAD

**Issue by nickgriffiths** [GoogleChrome/lighthouse-ci#481]
When I run `lhci autorun` on my sample project (contained here: [lighthouse-preconnect-error.tar.gz](https://github.com/GoogleChrome/lighthouse-ci/files/5459477/lighthouse-preconnect-error.tar.gz)) it fails with the following error:

```
  ✘  uses-rel-preconnect failure for maxLength assertion
       Preconnect to required origins
       https://web.dev/uses-rel-preconnect/
       Audit did not produce a value at all. "maxLength" might not be a valid assertion for this audit.

        expected: <=0
           found: NaN
      all values: NaN, NaN, NaN
```

The [Lighthouse Report](https://github.com/GoogleChrome/lighthouse-ci/files/5459511/Lighthouse.Report.html.zip) shows "Preconnect to required origins" as green, so I can't work out what the error is (if there is one). I did not expect to see `found: NaN`.

I can work around this by setting `"uses-rel-preconnect": "error"` in my config.

Background info: the project from which this sample was derived downloads the [Bazaarvoice loader](https://knowledge.bazaarvoice.com/wp-content/conversations/en_US/Display/display_integration.html#step-1-add-the-bv-loader), which then downloads resources from the URLs specified in the preconnect links.

**Comment by patrickhulce** [comment]
Thanks for filing @nickgriffiths! I know what's going on here I'll try to get a fix out today 👍

**Comment by patrickhulce** [comment]
published the fix in 0.6.0, please let me know if something is still not working as expected!
